### PR TITLE
chore: Setting up eslint and husky in client as well as server

### LINKED ISSUE #3: Setup eslint , husky

### PR DESCRIPTION
**Pull request by adityavyas611**
# Description
Setting up of Eslint (Airbnb style) in both client and server.Husky setup is also done using link-staged.

Fixes #3 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [x] Setting up the configuration files

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+	"singleQuote": true
```
Comment (nimish-gupta) [inline .prettierrc]: Spacing? Also add new line with every file, to keep it consistent with other files.

Comment (adityavyas611) [reply .prettierrc]: Done with the changes

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        2,
+        {
+          "allow": [
+            "_id"
+          ]
+        }
+      ]
+    }
+  }
+
```
Comment (ahhda) [inline .eslintrc.json]: Can we have a newline here as well? Thanks!

Comment (adityavyas611) [reply .eslintrc.json]: @ahhda The request changes has been performed, kindly approve.

**Comment by adityavyas611** [comment]
@ahhda , Nimish told us to squash the commits after PR review, so you can the commits, so once the PR will be approve i will squash the commits

**Pull request merged**